src/System.Net.Primitives/src/System/Net/Cookie.cs

```diff
@@ -10,6 +10,7 @@
 
 namespace System.Net
 {
+    [System.Runtime.CompilerServices.TypeForwardedFrom("System, Version=4.0.0.0, PublicKeyToken=b77a5c561934e089")]
```

Comment:
This one wouldn't require the manual System.dll shim as it is public so it should already be there.